ultrack/core/export/_test/test_exporter.py

```diff
@@ -40,3 +42,28 @@ def test_exporter(tracked_database_mock_data: MainConfig, tmp_path: Path) -> Non
         assert (tmp_path / f"tracks{file_ext}").stat().st_size > 0
 
         assert last_modified_time[str(tmp_file)] != tmp_file.stat().st_mtime
+
+
+def test_geff_zarr_extension_specific(
+    tracked_database_mock_data: MainConfig, tmp_path: Path
+) -> None:
+    """Test specific functionality of .geff.zarr extension in exporter."""
+    geff_file = tmp_path / "tracks.geff.zarr"
```

Comment:
```suggestion
    geff_file = tmp_path / "tracks.geff"
```
If the other suggestion is accepted.